packages/ethers/src/client.ts

```diff
@@ -924,6 +919,39 @@ export class Web3Modal extends Web3ModalScaffold {
     }
   }
 
+  private syncConnectedWalletInfo() {
+    const currentActiveWallet = window?.localStorage.getItem(EthersConstantsUtil.WALLET_ID)
+    const providerType = EthersStoreUtil.state.providerType
+
+    if (providerType === ConstantsUtil.EIP6963_CONNECTOR_ID) {
```

Comment:
Seems like there is no else state for the inner condition;
```suggestion
    if (providerType === ConstantsUtil.EIP6963_CONNECTOR_ID && currentActiveWallet) {
```